src/components/ThemeToggle.js

```diff
@@ -0,0 +1,25 @@
+import styles from "./themeToggle.module.scss";
+
+export default function ThemeToggle({ lightTheme, setLightTheme }) {
```

Comment:
Right now your component is too much specific. Let's abstract it a bit in order to make it more stupid and reusable.
Please change lightTheme to defaultValue, setLightTheme to toggle
Create inner state toggleValue
Change the inner state on click and call prop function toggle with new value